share/picongpu/pypicongpu/template/include/picongpu/param/png.param.mustache

```diff
@@ -35,18 +37,21 @@ namespace picongpu
 
     namespace visPreview
     {
-// normalize EM fields to typical laser or plasma quantities
-//-1: Auto:    enable adaptive scaling for each output
-// 1: Laser:   typical fields calculated out of the laser amplitude
-// 2: Drift:   typical fields caused by a drifting plasma
-// 3: PlWave:  typical fields calculated out of the plasma freq.,
-//             assuming the wave moves approx. with c
-// 4: Thermal: typical fields calculated out of the electron temperature
-// 5: BlowOut: typical fields, assuming that a LWFA in the blowout
-//             regime causes a bubble with radius of approx. the laser's
-//             beam waist (use for bubble fields)
+        // normalize EM fields to typical laser or plasma quantities
+        //-1: Auto:     enable adaptive scaling for each output
+        // 1: Laser:    typical fields calculated out of the laser amplitude
+        // 2: Drift:    [outdated]
+        // 3: PlWave:   typical fields calculated out of the plasma freq.,
+        //              assuming the wave moves approx. with c
+        // 4: Thermal:  [outdated]
+        // 5: BlowOut:  typical fields, assuming that a LWFA in the blowout
+        //              regime causes a bubble with radius of approx. the laser's
+        //              beam waist (use for bubble fields)
+        // 6: Custom:   user-provided normalization factors via customNormalizationSI
+        // 7: Incident: typical fields calculated out of the incident field amplitude,
+        //              uses max amplitude from all enabled incident field profile types ignoring Free
 {{#laser}}
-#define EM_FIELD_SCALE_CHANNEL1 1
+#define EM_FIELD_SCALE_CHANNEL1 7
```

Comment:
I am not sure what happens if we define no laser with PICMI. In that case it could be that value `7` is not valid.
@BrianMarre can PICMI create a setup without a laser?

Reply:
yes it can, unless I made an error somewhere.

And regarding your implied question, in this case it works due to the following
```
{{#laser}} // if a laser exist write the following
#define EM_FIELD_SCALE_CHANNEL1 7
{{/laser}}
{{^laser}} // if no laser exists write the following
#define EM_FIELD_SCALE_CHANNEL1 -1
{{/laser}}
```